tests/test_tomography_gate_set_tomography.py

```diff
@@ -0,0 +1,402 @@
+import numpy as np
+import pytest
+
+import qibo
+from qibo import gates
+from qibo.noise import DepolarizingError, NoiseModel
+from qibo.tomography.gate_set_tomography import *
```

Comment:
this line is what is making precommit fail. The imports must be done individually, as it was befofe.

Reply:
Oh I see, thanks so much for the information. I will be careful for the next PR.